modules/nw-running-dpdk-rootless-tap.adoc

```diff
@@ -0,0 +1,274 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/using-dpdk-and-rdma.adoc
+
+:_content-type: PROCEDURE
+[id="nw-running-dpdk-rootless-tap_{context}"]
+= Using the TAP CNI to run a rootless DPDK workload with kernel access
+
+DPDK applications can use `virtio-user` as an exception path to inject certain types of packets, such as log messages, into the kernel for processing. For more information about this feature, see link:https://doc.dpdk.org/guides/howto/virtio_user_as_exception_path.html[Virtio_user as Exception Path].
+
+In OpenShift Container Platform version 4.14 and later, you can use non-privileged pods to run DPDK applications alongside the tap CNI plugin. To enable this functionality, you need to mount the `vhost-net` device by setting the `needVhostNet` parameter to `true` within the `SriovNetworkNodePolicy` object.
+
+.DPDK and TAP example configuration
+image::rootless_dpdk_example.png[DPDK and TAP plugin]
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You are logged in as a user with `cluster-admin` privileges.
+* Ensure that `setsebools container_use_devices=on` is set as root on all nodes.
++
+[NOTE]
+====
+Use the Machine Config Operator to set this SELinux boolean.
+====
+
+.Procedure
+
+. Create a namespace `test-namespace` with the labels as in the following example YAML file, `test-namespace.yaml:
```

Comment:
do we do this in other examples of this section?

Reply:
I would say it's better to remove it to be consistent with other examples in this page, but this particular case is different. We need to explain why a rootless application needs to have the pod security privileged. 

https://kubernetes.io/docs/concepts/security/pod-security-standards/

You can say that it is needed because of the used capabilities and allowPrivilegeEscalation equal to true.